### ISSUE TITLE
shrink-image error

### ISSUE THREAD

**Issue by gsingh93** [radxa-repo/rbuild#15]
The `TARGET_BLOCKS` variable here ends up being an empty string for me, which causes more errors later: https://github.com/radxa-repo/rbuild/blob/main/rbuild#L90

This seems to be because `e2fsck` needs to be run first:
```
resize2fs 1.47.0 (5-Feb-2023)
Please run 'e2fsck -f /dev/mapper/loop11p2' first.
```

After running that, `shrink-image` works fine.

**Comment by RadxaYuntian** [comment]
I used to think `kpartx` can't print the correct loop device after attaching the file. Well I just tried again and it seems to work. I'll update the code so it is not guessing the device name.

**Comment by RadxaYuntian** [comment]
Closed by d3e949fd625d10d1ddefa76945dcf2612a1ffa85.
